pom.xml

```diff
@@ -250,7 +250,6 @@
             <groupId>software.amazon.awssdk.crt</groupId>
             <artifactId>aws-crt</artifactId>
             <version>0.29.18</version>
-            <classifier>fips-where-available</classifier>
```

Comment:
I think you should just revert this https://github.com/aws-greengrass/aws-greengrass-nucleus/commit/3fcb326479227a6c2492ce06648f81a15c6d5e03 - revert the exclusion of crt in sdk snapshot

Reply:
I will update PR to revert only the exclusion of the CRT